lib/components/Layout/Header.js

```diff
@@ -15,43 +15,48 @@ import styles from './Header.css';
  *
  * @param {string} param0.logo logo url
  * @param {string} param0.company company name
+ * @param {function} param0.logoRenderer render prop
  */
-function Branding({ logo, company }) {
+function Branding({ logo, company, logoRenderer }) {
+  let content;
   if (typeof logo === 'string') {
-    return <img className={styles.brandingLogo} src={logo} alt={company} />;
-  }
-  if (logo === false) {
-    return <span className={styles.brandingText}>{company}</span>;
+    content = <img className={styles.brandingLogo} src={logo} alt={company} />;
+  } else if (logo === false) {
+    content = <span className={styles.brandingText}>{company}</span>;
+  } else {
+    const Logo = logo;
+    content = (
+      <div className={styles.brandingLogo}>
+        <Logo height={20} />
+      </div>
+    );
   }
-  const Logo = logo;
-  return (
-    <div className={styles.brandingLogo}>
-      <Logo height={20} />
-    </div>
-  );
+  return logoRenderer ? logoRenderer(content) : content;
 }
 
 Branding.displayName = 'Branding';
 
 Branding.propTypes = {
   company: PropTypes.string,
   logo: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
+  logoRenderer: PropTypes.func,
 };
 
 Branding.defaultProps = {
   company: 'Virtru',
   logo: logoWhite,
+  logoRenderer: undefined,
```

Comment:
maybe will be better to use null?

Reply:
null means that value still exists, while undefined tells that value is not specified at all